src/main/java/com/minecolonies/coremod/tileentities/ScarecrowTileEntity.java

```diff
@@ -578,7 +518,7 @@ public void read(final CompoundNBT compound)
             final ItemStack stack = ItemStack.read(inventoryCompound);
             if (ItemStackUtils.getSize(stack) <= 0)
             {
-                inventory.setStackInSlot(i, ItemStackUtils.EMPTY);
+                inventory.setStackInSlot(i, ItemStack.EMPTY);
```

Comment:
usage of `ItemStackUtils` version is preferred for easier future porting (reason it was made in the first place)